components/src/jsMain/kotlin/dev/fritz2/components/box.kt

```diff
@@ -33,6 +33,13 @@ import dev.fritz2.styling.params.GridParams
  * @param prefix the prefix for the generated CSS class resulting in the form ``$prefix-$hash``
  * @param content a lambda expression for setting up the content and events of the ``div`` element itself
  */
+@Deprecated(
```

Comment:
I think the replacement expression is far too complicated and wrong:
```kotlin
box({
    background { color { "purple" } }
}) {
    +"Test"
}
```
If applied the above becomes to:
```kotlin
div(fun FlexParams.() {
    background { color { "purple" } }
}, StyleClass.None, null, "box") {
    +"Test"
}
```
Definitly not correct!

Should it not simplex be a chnage from `box` to `div` like this?
```kotlin
div({
    background { color { "purple" } }
}) {
    +"Test"
}
```
The import is nice of course!

Reply:
Afaik `ReplaceWith` does not have full support for both optional parameters and lambdas which results in the generated replacement being overly complicated, yet technically correct. The only other option that comes to my mind is to get rid of the `ReplaceWith` as the conversion is simple enough to just be mentioned in the release notes 🤔